### DISCUSSION TITLE
Relay status indicator

### DISCUSSION THREAD

**Discussion by summongod** [Blackymas/NSPanel_HA_Blueprint#337] [General]
I am using one button to control a set of lights via the hardware relay and the second to toggle a button entity that control a set of LEDs. The blue indicator on the screen works for the relay but not for the entity control. Is there a way for me to configure it to do so?

**Comment by Blackymas** [comment]
Do not understand the question. The blue line shows the status of the entity that is stored in the blueprint.

For the relays, the icon with 1 (Releay 1) and the icon with 2 (Releay 2) are displayed at the top of the top menu.

Both are separated by an asterisk and are displayed appropriately.

**Reply by summongod** [reply]
I can't get the blue line to display on a switch entity. I have solved by using a light entity helper instead. This works as expected.
Thanks.

**Comment by deejaybeam** [comment]
> I am using one button to control a set of lights via the hardware relay and the second to toggle a button entity that control a set of LEDs. The blue indicator on the screen works for the relay but not for the entity control. Is there a way for me to configure it to do so?

Thats a misconfig from your side I guess.
The one button with lights assigned could of course display the state of that lights (on/off)
But if you assign the hw-button to a (helper) button and that controls lights via script/automation?? how do you think that the display should get the knowledge of a state to display?
You could create a helper-group of type light to group some lights and assign that directly to the hw-button - done

**Reply by summongod** [reply]
That's a good solution. I was trying to use Switch as X (light) helper because my leds are controlled by a switch template exported from another device in ESPhome. This doesn't work. I was also fooled by the display toggling the indicator when any automation is assigned to the button.
Thanks
